windows/src/desktop/kmshell/xml/strings.xml

```diff
@@ -1193,7 +1193,7 @@ PRIVACY WARNING: Please note that the debug logfile records all keystrokes that
   <!-- Context: OSK Font Helper -->
   <!-- String Type: FormatString -->
   <!-- Introduced: 8.0.294.0 -->
-  <string name="S_OSK_FontHelper_PleaseWait" comment="OSK Font helper message">Please wait while searching for releted fonts for keyboard %0:s</string>
+  <string name="S_OSK_FontHelper_PleaseWait" comment="OSK Font helper message">Please wait while searching for related fonts for keyboard %0:s</string>

```

Comment:
Ran a quick spell check, and found typos in comments that could be fixed (while in this file)

l. 442
permissible

l. 474
system tray